elements/k-new-post.html

```diff
@@ -24,68 +51,106 @@
       }
 
       .preview {
+        padding-left: 15px;
+        padding-right: 15px;
         font-size: 18px;
         line-height: 1.4em;
       }
     </style>
+    <iron-media-query query="(min-width: 752px)" query-matches="{{wide}}"></iron-media-query>
 
     <paper-tabs selected="{{selected}}">
       <paper-tab>Edit</paper-tab>
       <paper-tab>Preview</paper-tab>
     </paper-tabs>
 
-    <iron-pages selected="{{selected}}">
+    <iron-pages selected="{{selected}}" wide-layout$="{{wide}}">
```

Comment:
@duykhoa That's my fault! I follow demo code of iron-query-media element. After that, I forgot remove it. I'm so sorry about that :cry:  I'll remove it now! 
Next time I'll be more careful!